lib/components/util/dropdown.tsx

```diff
@@ -161,7 +161,7 @@ const Dropdown = ({
       id={`${id}-wrapper`}
       onKeyDown={_handleKeyDown}
       ref={containerRef}
-      role="group"
+      role="listitem"
```

Comment:
Using role='listitem' is also, unfortunately, [not ideal](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles#:~:text=available%20and%20supported.-,Avoid%20using%3A,-application)

Reply:
It's an `li` anyway! Thanks for the catch